install_conda_deps.sh

```diff
@@ -4,4 +4,6 @@ conda install -n $YOURENVNAME requests=2.13.0
 conda install -n $YOURENVNAME -c menpo opencv=2.4.11
 conda install -n $YOURENVNAME pyqt=5.6.0
 conda install -n $YOURENVNAME qtawesome=0.4.4
+conda install -n $YOURENVNAME nomkl numpy scipy scikit-learn numexpr
+conda remove -n $YOURENVNAME mkl mkl-service
```

Comment:
Lets do each of these on their own line with a version number. Also why do we remove mkl from conda? Is it possible to just suppress it from being added instead?